docs/examples/cloudbuildsource/README.md

```diff
@@ -90,12 +90,12 @@ by looking at the logs of the service that this CloudBuildSource sinks to.
 Validation: valid
 Context Attributes,
   specversion: 1.0
-  type: com.google.cloud.build.event
+  type: google.cloud.cloudbuild.build.v1.statusChanged
   source: //cloudbuild.googleapis.com/projects/PROJECT_ID/builds/BUILD_ID
   subject: SUCCESS
   id: 1085069104560583
   time: 2020-04-03T23:51:29.811Z
-  dataschema: https://raw.githubusercontent.com/google/knative-gcp/master/schemas/build/schema.json
+  dataschema: https://github.com/googleapis/google-cloudevents/blob/master/proto/google/events/cloud/scheduler/v1/data.proto
```

Comment:
The schema is wrong. 
In addition, the path of schema may change because of this PR https://github.com/googleapis/google-cloudevents/pull/57. I think this PR should get merged soon.